using_images/other_images/jenkins_slaves.adoc

```diff
@@ -86,7 +86,7 @@ $ docker pull openshift/jenkins-slave-base-centos7
 $ docker pull openshift/jenkins-slave-maven-centos7
 $ docker pull openshift/jenkins-slave-nodejs-centos7
 $ docker pull openshift/jenkins-agent-maven-35-centos7
-$ docker pull openshift/jenkins-agent-nodejs-8-centos7
+$ docker pull openshift/jenkins-agent-nodejs-12-centos7
```

Comment:
And `openshift/jenkins-agent-nodejs-10-centos7`

Reply:
Thank you @siamaksade I will make these changes, I have no confirmation from @akram on changes in 4.X versions of the doc. Considering that .8 is EOL I think it makes sense to change it in 4.X too, WDYT?

Reply:
Agreed